### ISSUE TITLE
kcov error with alias

### ISSUE THREAD

**Issue by Mark-E-Hamilton** [SimonKagstrom/kcov#57]
I have this simple test bash script:

```
#!/bin/bash
shopt -s expand_aliases
alias test_alias='{
echo called test_alias
} '
test_alias
````
When I run kcov on it, I get this output
```
% kcov /tmp/mhamilt/kcov_out ./test.sh
called test_alias
echo called test_alias
} '
````
Why is kcov printing everything after the first line of the alias definition?  That alias syntax is valid (or at least clearly works in the test code and in our larger code base.)  In our larger case the alias is many lines long; all but the first are printed.

BTW, this is kcov-25 on a Linux RHEL6 machine, using bash-4.1.2.

(An unrelated question; why does kcov not have a --version/-v option?)

Thanks.

Mark E. Hamilton

**Comment by SimonKagstrom** [comment]
The root cause is that kcov will re-print everything on stderr which doesn't have a magic tag at the start. An example with bash -x shows how this looks for the alias test:

```
$ PS4='kcov@${BASH_SOURCE}@${LINENO}@' bash -x tests/bash/multiline-alias.sh 
kcov@tests/bash/multiline-alias.sh@2@shopt -s expand_aliases
kcov@tests/bash/multiline-alias.sh@5@alias 'test_alias={
echo called test_alias
} '
kcov@tests/bash/multiline-alias.sh@6@echo called test_alias
called test_alias
```

So in this case, kcov would need to identify the '...' quotes to denote a string (and not just see them as valid stderr output).

**Comment by SimonKagstrom** [comment]
Fixed with 55a77e1. Ugly fix though...
